src/blockchain/Blockchain.ts

```diff
@@ -174,50 +175,62 @@ export default class Blockchain implements IBlockchainApi {
   /**
    * Pushes a new [[Accumulator]] on chain.
    *
-   * @param address The chain address of the [[Attester]].
    * @param accumulator The new [[Accumulator]].
+   * @returns Returns an object that can be used to submit a transaction.
    */
-  public async updateAccumulator(
-    address: KeyringPair,
+  public updateAccumulator(
     accumulator: Accumulator
+  ): SubmittableExtrinsic<'promise'> {
+    return this.api.tx[this.chainmod].updateAccumulator(accumulator.toString())
+  }
+
+  /**
+   * Signs and sends a transaction to a blockchain node.
+   * The return promise will resolve if the transaction was included in a finalized block.
+   * If the transaction errors, the promise will be rejected.
```

Comment:
Docs: _errors_ is not a verb, I propose to replace it with _fails_.

Reply:
done!